### DISCUSSION TITLE
[Feature] Date format selection

### DISCUSSION THREAD

**Discussion by E440QF** [immich-app/immich#6132] [Feature Request]
### The feature

Currently there doesn't seem to be an option to change the format in which dates are shown in the UI, i don't believe that's strictly necessary in places like image information where it's month dd, yyyy but for example in the birthday selection window (WEB) there is an input field accepting mm/dd/yyyy i'm not sure if there's any other occurencies of that format but there should be a setting for choosing how numerical dates are handled

![Screenshot_20240102_191943](https://github.com/immich-app/immich/assets/42470585/e889738b-43a2-4b71-85f0-88d03a42fb70)

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by fedepujol** [comment]
I need my dd/mm/yyyy picker selector

**Comment by Gener8fotos** [comment]
Yes please add this feature, i need my photos to follow and be searchable with the international standard ISO8601 (YYYY-MM-DD HH:MM)
this is because all my devices (phones, cameras, etc) all use this format since 1997

**Comment by koskit** [comment]
As a side note, make this globally editable. There is another place it exists, when changing the date of a photograph. 
I need to manually edit some dates on some scanned photographs, and for people that do not have mm/dd/yyyy, it's counter intuitive. I keep picking the date first.

**Comment by bigntallmike** [comment]
Is there any progress on this? The date interface is entirely non-intuitive to a lot of people out there, myself included.
Can we please move to ISO standard dates at least? If you're going to irritate people, at least do it in an internationally approved way.

**Comment by githubsean** [comment]
I found this same issue on the date search.

<img width="1024" height="2284" alt="Screenshot_20251008-085250" src="https://github.com/user-attachments/assets/961e2e56-5324-4749-b6a6-e7f0412d9907" />

Honestly, I'd call this a bug. I'm in Australia and my Android system locale is set to Australia. Like the entire rest of the world, we use dd/mm/yyyy.

**Comment by geoleven** [comment]
I would add that if there should be one standard something more neutral should be the one. I understand that probably the initial author is usa based according to the choice but then the best one is indeed YYYY/MM/DD (and is also easier to sort for).

**Reply by bigntallmike** [reply]
https://en.wikipedia.org/wiki/ISO_8601 describes the ISO standard date format as YYYY-MM-DD its not just a preference, its a standard.
